pilz_trajectory_generation/src/path_circle_generator.cpp

```diff
@@ -41,15 +41,14 @@ std::unique_ptr<KDL::Path> PathCircleGenerator::circleFromCenter(
   KDL::RotationalInterpolation* rot_interpo = new KDL::RotationalInterpolation_SingleAxis();
   try
   {
-    KDL::Path* circ = new KDL::Path_Circle(start_pose,
+    return std::unique_ptr<KDL::Path>(new KDL::Path_Circle(start_pose,
```

Comment:
`std::make_unique<..>(..)`?

Reply:
c++11 ...

Reply:
https://en.cppreference.com/w/cpp/memory/unique_ptr/make_unique

https://stackoverflow.com/questions/7038357/make-unique-and-perfect-forwarding

Could bring/roll your own.

Reply:
Ah, I didn't know that. Thanks for the hint.